### PR TITLE
cmd: ensures valid network during create dkg

### PR DESCRIPTION
**Pull request by dB2510**
Ensures valid network during create dkg using network flag which is then used to get fork version to create cluster definition.

category: bug
ticket: #583

### CODE REVIEW

File: dkg/dkg.go [hunk 1]
```diff
@@ -67,6 +66,11 @@ func Run(ctx context.Context, conf Config) (err error) {
 		return err
 	}
 
+	network, err := forkVersionToNetwork(def.ForkVersion)
```
Comment (corverroos) [inline dkg/dkg.go:69]: I think we should do it the other way round, provide a network and map it to fork version. Requiring people to input fork versions is not a great UX.

Comment (xenowits) [reply dkg/dkg.go:69]: agreed

Comment (dB2510) [reply dkg/dkg.go:69]: Done!
Modified in createdkg command flags

**Review by xenowits — APPROVED** [review]
LGTM🚀

File: dkg/dkg.go [hunk 2]
```diff
@@ -565,3 +569,20 @@ func waitConnect(ctx context.Context, tcpNode host.Host, p peer.ID) (time.Durati
 
 	return 0, false
 }
+
+func forkVersionToNetwork(forkVersion string) (string, error) {
```
Comment (xenowits) [inline dkg/dkg.go:573]: @corverroos do you think functions like `forkVersionToNetwork()` and `networkToForkVersion()` should be separated out somewhere?

Comment (dB2510) [reply dkg/dkg.go:573]: This is very small piece of code, can be duplicated imo

Comment (corverroos) [reply dkg/dkg.go:573]: Indeed, also remember the rule of 3, only once it is used in 3 places or more, is it worth extracting.

**Review by corverroos — APPROVED** [review]

**Pull request merged**